dashboard/test/ui/features/star_labs/dance/dance_party.feature

```diff
@@ -68,18 +68,20 @@ Feature: Dance Party
 
     When I navigate to the shared version of my project
     And element ".signInOrAgeDialog" is hidden
+    # Tested level from Dance Party 2018 level 13 does not use visualization column preview.
```

Comment:
Nit: it's not the "checking" of run/reset buttons that resolves flakiness, right? It's running it once gets us into a consistent state? Something like:

```
We run/reset here once before checking the number of sprites below,
which helps resolve some not fully understood test flakiness related to Dance Party's "preview mode".
The level used in this test (Dance 2018, level 13) does not use preview mode,
[but sometimes ends up in preview mode sometimes when shared I don't understand this part :)]
```

Reply:
@bencodeorg - How does this sound?
```
    # We run/reset here once before checking the number of sprites below which helps
    # resolve some not fully understood test flakiness related to Dance Party's "preview mode".
    # The level used in this test (Dance Party 2018, level 13) does not use preview mode but
    # when level is shared, the standalone Dance project level does use preview mode.
```